docs/editors-guide/rare-disease-curator.md

```diff
@@ -0,0 +1,51 @@
+# Rare Disease Curation
+
+Rare diseases in Mondo are created based on 3 main subsets. Rules for editing these subsets depends on the subset type. Before editing rare disease information in Mondo, please familiarize yourself with how rare diseases are represented in Mondo [here](https://mondo.readthedocs.io/en/latest/editors-guide/rare-disease-subset/).
+
+It is important to remember that the rare disease subsets are updated automatically at the time of the monthly Mondo release. It is, therefore, crucial that editing rare disease information is done by following the instructions below for these edits to be retained after the release.
+
+**1. Rare disease from authority sources.**
+These terms have an `in_subset` annotation `X_rare`  (where X is the rare disease authority).
+These subsets are maintained automatically:
+- a list of rare diseases is maintained for each source here:
+    - src/ontology/subsets/nord-subset.template.tsv
+    - src/ontology/subsets/gard-subset.template.tsv
+- during the Mondo release, the `X_rare` subset is updated based on the Mondo terms on the authoritative source list
+
+Until we have set a process with the sources to maintain this list automatically, Mondo editors should update this list of rare diseases manually **upon request by the authoritative source**.
```

Comment:
@matentzn is that right?

Reply:
Yes, right now:

NORD and GARD are managed here: https://github.com/monarch-initiative/mondo/tree/master/src/ontology/subsets

Orphanet is managed in mondo-edit.obo